crates/ide/src/inlay_hints/bind_pat.rs

```diff
@@ -818,6 +818,7 @@ fn fallible() -> ControlFlow<()> {
     }
 
     #[test]
+    #[ignore]
```

Comment:
```suggestion
    #[ignore] // FIXME(salsa-transition): bring this back
```

Reply:
decided to unignore this and accept the weird IDs for now. they seem mostly stable?